content-org/posts.org

```diff
@@ -0,0 +1,150 @@
+
+	  You are going to push your existing local repo to this new remote repo, so your can ignore the part about README and .gitignore like it says on the page.
+
+	  GitHub should take you to this page after the repo is created.
+	  #+attr_html: :alt GitHub new repo UI
+	  #+attr_html: :width 600
+	  [[file:github_new_repo.png]]
+
+	  Following the instruction in "…or push an existing repository from the command line", run
+
+	  #+BEGIN_SRC sh
+	  $ git remote add origin https://github.com/<username>/<repo_name>.git
+	  $ git push -u origin master
+	  #+END_SRC
+
+	  from your local repo directory to push it to your GitHub remote repo. Make sure you are on the master branch in your local repo when running this command.
+
+***** Configuring GitHub page
+
+	  Now that your code is on GitHub, it's time to spin up a GitHub page.
+
+	  Go to your repo's settings section from its homepage.
+	  #+attr_html: :alt GitHub repo UI
+	  #+attr_html: :width 600
+	  [[file:github_repo_home.png]]
+
+	  Scroll down to the GitHub Pages section.
+
+	  You should see something like this.
+	  #+attr_html: :alt GitHub repo UI
+	  #+attr_html: :width 600
+	  [[file:github_page_none.png]]
+
+	  You want to change the Source drop-down to "master branch /docs folder". And this is why I asked you to export your game to the docs folder earlier. You could just have your web game files in your PROJECT_ROOT directory and change the Source drop-down to "master branch", but I prefer the docs method because it makes your project more organized in my opinion.
+
+	  And that's it, you are done! Your game should now be accessible from the URL indicated in the GitHub Pages section.
+
+	  Now when you make changes to your game and makes a new build, just push your updated master branch to GitHub and it will serve the new version of your game.
+
+***** Extra thoughts
+
+	  Although this post is written specifically for hosting Godot game on GitHub, the same workflow should work for any engine that exports to web format and any source control site that has something similar to GitHub pages.
+
+	  For instance, I believe Unity has a option to export with WebGL and Bitbucket Cloud seems to be Bitbucket's version of GitHub pages. 
+
+	  Also, there are many ways to refine the workflow described in this post in case you want to push your game beyond the game jam sphere.
+
+	  Things like, using git submodule to keep your game code and exported files in their own repos so you don't have to use the docs folder of the code repo. Or incorporating automated testing since GitHub works really well with some of the continuous integration(CI) services out there. GitHub is even starting to implement its own CI workflow so you don't have to use other tools/services. Check out [[https://github.com/features/actions][GitHub Actions]] if you are curious.
```

Comment:
> There are also ways to refine the workflow described here in case you want to push your game beyond the game jam sphere. 

> Using git submodules to keep your game code and exported files in their own repos so you don't have to use the _docs_ folder of the code repo, or incorporating automated testing since GitHub works really well with some of the continuous integration (CI) services available, including GItHub's own [[https://github.com/features/action][GitHub Actions]].